### ISSUE TITLE
Uncaught TypeError: d3.select(...).classed(...).attr is not a function

### ISSUE THREAD

**Issue by lgn21st** [frairon/go-outline#44]
[Enter steps to reproduce below:]
1. ...
2. ...

**Atom Version**: 1.8.0
**System**: Mac OS X 10.11.5
**Thrown From**: [go-outline](https://github.com/frairon/go-outline) package, v0.4.0
### Stack Trace

Uncaught TypeError: d3.select(...).classed(...).attr is not a function

```
At /Users/Daniel/.atom/packages/go-outline/lib/go-outline-view.coffee:368

TypeError: d3.select(...).classed(...).attr is not a function
  at HTMLLIElement.updateExpanderIcon (/Users/Daniel/.atom/packages/go-outline/lib/go-outline-view.coffee:368:40)
  at Selection.selection_each [as each] (/Users/Daniel/.atom/packages/go-outline/node_modules/d3/node_modules/d3-selection/build/d3-selection.js:536:39)
  at createChildren (/Users/Daniel/.atom/packages/go-outline/lib/go-outline-view.coffee:380:12)
  at /Users/Daniel/.atom/packages/go-outline/lib/go-outline-view.coffee:417:38
  at Selection.selection_call [as call] (/Users/Daniel/.atom/packages/go-outline/node_modules/d3/node_modules/d3-selection/build/d3-selection.js:500:14)
  at GoOutlineView.module.exports.GoOutlineView.updateSymbolList (/Users/Daniel/.atom/packages/go-outline/lib/go-outline-view.coffee:417:26)
  at GoOutlineView.updateSymbolList (/Users/Daniel/.atom/packages/go-outline/lib/go-outline-view.coffee:1:1)
  at /Users/Daniel/.atom/packages/go-outline/lib/go-outline-view.coffee:159:26

```
### Commands

```
     -0:06.9.0 go-outline:toggle (atom-text-editor.editor.is-focused.vim-mode.normal-mode)
```
### Config

``` json
{
  "core": {
    "customFileTypes": {
      "source.ini": [
        ".buckconfig",
        ".flowconfig"
      ],
      "source.json": [
        ".arcconfig",
        "BUCK.autodeps"
      ],
      "source.python": [
        "BUCK"
      ]
    },
    "disabledPackages": [
      "tree-view"
    ],
    "projectHome": "/Users/Daniel/workspace"
  },
  "go-outline": {
    "showTree": false
  }
}
```
### Installed Packages

``` coffee
# User
editorconfig, v1.4.1
go-outline, v0.4.0
haskell-grammar, v0.4.0
highlight-selected, v0.11.2
language-babel, v2.29.2
language-ini, v1.16.0
language-ocaml, v1.1.2
language-thrift, v1.0.2
merge-conflicts, v1.4.4
nuclide, v0.148.0
tool-bar, v0.4.0
vim-mode, v0.65.1

# Dev
No dev packages
```

**Comment by frairon** [comment]
Hi lgn21st,
thanks for taking the time to report this!
I think the problem is that I did not specify a d3 version in the package.json which then made npm install d3 version 4 which changed quite a lot.
I fixed the d3 version to 3.5.17 and since then, cannot reproduce the error.
Please verify if you still get the issue.
Thank you!

**Comment by lgn21st** [comment]
Thank you, the report is auto-generated by Atom ;-)
